frontend/src/layouts/EmptyLayout.tsx

```diff
@@ -0,0 +1,29 @@
+import './PublicLayout.scss';
+
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import { Footer, EmptyHeader } from 'components/layout';
+
+const EmptyLayout: React.FC = ({ children }) => {
```

Comment:
simply layout with no context hooks, can be displayed without loading keycloak, redux, etc.